### ISSUE TITLE
Repacking skips bk2 files]

### ISSUE THREAD

**Issue by jvarnes** [rfuzzo/CP77Tools#39]
My dreams of a no intro movies mod came to a crashing halt when packing skipped all the bk2 files from basegame_5_video.archive :(

[Error][12/23/2020 4:43:38 AM] Failed to read cr2w file F:\CPPTool\basegame_5_video\base\movies\fullscreen\common\cyberpunk2077_game_intro.bk2

Looking for the cr2w header instead of KB2j on this file type?

**Comment by Absolucy** [comment]
Also repacking screws up any audio, even if it's all untouched (just straight unpack -> repack)

**Comment by jvarnes** [comment]
Not that this fixes the bug, but swapping out the intro biks prolly won't do what I want anyway as I was able to cut them off in the inkwidget file and the game still has a large pause before starting so until we can actually decode and encode crw2 files it's probably moot.

Anyway the pack function needs to look at the headers of the files it's adding for bik and other support (like those audio files wem I'm guessing? or whichever format they are)

**Comment by rfuzzo** [comment]
Yes correct, packing only supports cr2w files right now (it does not support audio or video) 
Thank you for the report

**Comment by jvarnes** [comment]
Thanks fuzzo! Keep up the good work!

**Comment by rfuzzo** [comment]
this issue is fixed here: https://github.com/WolvenKit/CP77Tools/releases/tag/v.0.1.4.0h2
